kim_tools/ase/core.py

```
@@ -933,263 +936,1064 @@ def energy_cheat(a, dimer, large_cell_len, offset, einf):
+    return [
+        {
+            "index": int(i),
+            "alat": float(scan["alats"][i]),
+            "energy_per_atom": float(scan["energies_per_atom"][i]),
+        }
+        for i in minima_indices
+    ]
+
+
+def query_kim_influence_distance(model_name: str) -> float:
+    units_accepted, kim_model = kimpy.model.create(
```

Ruff (0.15.20)

[warning] Unpacked variable `units_accepted` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)